js/lib/urlutil.js

```diff
@@ -429,8 +430,11 @@ const UrlUtil = {
     }
 
     let parsed = urlParse(location)
-    if (parsed.origin) {
-      // parsed.origin is specific to muon.url.parse
+    // parsed.origin is specific to muon.url.parse
+    if (parsed.origin !== undefined) {
+      if (parsed.protocol === 'about:') {
```

Comment:
for more backwards consistency, the conditional should be `if (parsed.origin === '') {..}` instead of checking if it is 'about:'. it seems like chromium's url parser will return '' as the origin for protocols that don't technically have origin-scoping (ex: 'data:') whereas previously this would return protocol + host.